src/transformers/utils/dummy_tf_objects.py

```diff
@@ -1350,6 +1350,13 @@ def __init__(self, *args, **kwargs):
         requires_backends(self, ["tf"])
 
 
+class TFGPT2Tokenizer(metaclass=DummyObject):
+    _backends = ["tf"]
+
+    def __init__(self, *args, **kwargs):
+        requires_backends(self, ["tf"])
```

Comment:
I don't think this one should exist 🤔 (the dummy import in the main `__init__.py` points to `utils/dummy_keras_nlp_objects.py`)

Reply:
Yeah you're right. Done.